CMakeLists.txt

```diff
@@ -39,7 +39,13 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 if (MSVC)
 	set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT CemuBin)
 	# floating point model: precise, fiber safe optimizations
-	add_compile_options(/EHsc /fp:precise /GT)
+	add_compile_options(/EHsc /fp:precise)
+	if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+		add_compile_options(/clang:-maes)
```

Comment:
What's the purpose of adding the -maes flag?

Reply:
I see it exists in src/CMakeLists.txt as well, so it makes sense to have it reflected here.
What requires it though?

Reply:
compiling used to fail because of missing aes intrinsics due to missing that flag
I just tested again and it no longer fails so I'm going to remove it.